### ISSUE TITLE
Request to update dependency on ember-get-config to 0.3.0

### ISSUE THREAD

**Issue by havremunken** [fenichelar/ember-simple-auth-token#298]
Hi there,

My ember-dependency-lint is angry and it is because this is the last library I use that uses the older 0.2.4 version.

Could I please request an update that uses 0.3.0, like many other packages do?

Thanks!

**Comment by fenichelar** [comment]
@havremunken Are you able to test `https://github.com/fenichelar/ember-simple-auth-token#master`?

**Comment by havremunken** [comment]
@fenichelar I replaced the official ember-simple-auth-token with your version, started the server, and I could log in / log out etc. as before. Seems to be working perfectly!

The dependency linter is also satisfied.

I have not been able to test the library in a deeper sense, because of some issues with running headless Chrome tests in Debian running under WSL2 on Windows, but everything I have been able to test, has passed.

I am keeping the branch in which I tested this around for a while if you have anything more concrete you want me to test. :)

Thanks for the quick response!
